python/py-libcloud/Portfile

```diff
@@ -71,3 +71,10 @@ if {${name} ne ${subport}} {
     livecheck.url   [lindex ${master_sites} 0]
     livecheck.regex ">${_name}-(\\d+(\\.\\d+)+)\\${extract.suffix}<"
 }
+
+foreach {old new} {34 36 35 36} {
+    subport py${old}-${python.rootname} "
+        replaced_by py${new}-${python.rootname}
+        PortGroup obsolete 1.0
+    "
+}
```

Comment:
nobody asked to put this back - the consensus a while ago was that these replacement rules are not useful (that was why the `py-graveyard` port was removed). This is just a historical leftover, the removal you did initially was a good one.

Reply:
:) OK, thanks. I'll remove that again, squash, and re-push.